nss-day-cohort-47/holiday-road-everglade-explorers
# Bv eateryitinerary

# Description

The event listener for the eateries is up and populating its data to the preview section of the index.html

Fixes # (issue)

## Type of change


# Testing Instructions

Serve the page and select an eatery from the dropdown. You should see the eatery name and the city and state in the third box of the preview.

# Checklist:
